tachyon/zk/lookup/log_derivative_halo2/prover.h

```diff
@@ -23,7 +23,7 @@ namespace tachyon::zk::lookup::log_derivative_halo2 {
 template <typename BigInt>
 struct TableEvalWithIndex {
   RowIndex index;
-  BigInt eval;
+  BigInt eval = BigInt::Zero();
```

Comment:
I think this might be a question for the future, but I'm wondering why we have a `BigInt::Zero()` and `BigInt::One()` if they just run `return BigInt(0);` and `return BigInt(1);` anyway. (Opt. & You can resolve this when read)
```suggestion
  BigInt eval(0);

Reply:
This should be in the previous commit, right?

Reply:
Removed. I don't think this is necessary since in previous commit all tests pass.